packages/devtools/lib/src/model/model.dart

```diff
@@ -99,9 +99,10 @@ class App {
     return framework.connectDialog.isVisible();
   }
 
-  Future<void> connectDialogConnectTo([dynamic port]) async {
+  Future<void> connectDialogConnectTo([dynamic uri]) async {
+    // uri comes as a String (from JSON) so needs changing back to a URI.
     // ignore: invalid_use_of_visible_for_testing_member
-    return framework.connectDialog.connectTo(port);
+    return framework.connectDialog.connectTo(Uri.parse(uri));
```

Comment:
not your fault but go ahead and make this method not claim to be visible for testing.

Reply:
There are three other instances of this - should I remove them all?

Reply:
Go ahead and remove any ones that aren't in test files.

Reply:
Done!